studio/data/subscriptions/org-subscription-update-mutation.ts

```diff
@@ -0,0 +1,62 @@
+import { useMutation, UseMutationOptions, useQueryClient } from '@tanstack/react-query'
+import { put } from 'lib/common/fetch'
+import { API_URL } from 'lib/constants'
+import { subscriptionKeys } from './keys'
+
+export type SubscriptionTier =
+  | 'tier_free'
+  | 'tier_pro'
+  | 'tier_payg'
+  | 'tier_team'
+  | 'tier_enterprise'
+
+export type OrgSubscriptionUpdateVariables = {
+  slug: string
+  paymentMethod?: string
+  tier: SubscriptionTier
+}
+
+export type OrgSubscriptionUpdateResponse = {
+  error?: any
+}
```

Comment:
Is `SupaResponse<T>` useful here at all?

Reply:
oh yeah that works too actually i think